src/Razor/test/Microsoft.AspNetCore.Razor.LanguageServer.Test/Formatting/CodeDirectiveFormattingTest.cs

```diff
@@ -70,7 +70,7 @@ public class Foo{
     public class Foo
     {
         void Method()
-        { 
```

Comment:
I would argue that the way this was before was wrong (can't leave trailing spaces), but if anyone has compelling reasons why that's not the case let me know.

Reply:
The trailing spaces is _definitely_ something I ran into with the old formatting that was frustratingly hard to fix (ie, I gave up), so not only is this better, its a pretty good validation for the benefits of your approach! 👍👍👍👍